model/model.go

```diff
@@ -83,6 +83,7 @@ type ColumnInfo struct {
 	types.FieldType     `json:"type"`
 	State               SchemaState `json:"state"`
 	Comment             string      `json:"comment"`
+	ExplictedCollation  bool        `json:"explicted_collation"`
```

Comment:
"Explicit" is an adjective not a verb, there's no need to add the -ed suffix.

```suggestion
	ExplicitCollation   bool        `json:"explicit_collation"`
```

Reply:
Done,thanks for your help.